spec/helpers/sufia_helper_spec.rb

```diff
@@ -27,6 +27,14 @@
     end
   end
 
+  describe "#link_to_field" do
```

Comment:
Could you add a test to make sure that the #to_s interface is used? Maybe pass an RDF::URI as contributor instead of a string?